common/containers/Tabs/SendTransaction/components/AmountField.jsx

```diff
@@ -65,7 +65,7 @@ export default class AmountField extends React.Component {
 
   onSendEverything = () => {
     if (this.props.onChange) {
-      this.props.onChange("everything", this.props.unit);
+      this.props.onChange('everything', this.props.unit);
```

Comment:
Oh yeah I like this component being more self sufficient. There's a lot of cleanup I want to do to send to make it less monolithic (For example, move the form into its own component so there's less `if (this.props.wallet)` going on.) Would you mind if I just cherry picked the commit in from #60 as a part of another cleanup PR?